jdaviz/configs/imviz/plugins/tools.py

```diff
@@ -153,7 +153,7 @@ def on_mouse_or_key_event(self, data):
         # Note that we throttle this to 200ms here as changing the contrast
         # and bias it expensive since it forces the whole image to be redrawn
         if event == 'dragmove':
-            if (time.time() - self._time_last) <= 0.2:
+            if (time.time() - self._time_last) <= 0.1:
```

Comment:
I wonder if throttling on the JS-side would make any difference on performance (in case there is any bottleneck in the callbacks themselves)?

Reply:
How would I do that? I can give it a try if you want. The throttle here only affects the mouse-drag event but I think there is a separate throttle for Plot Option sliders too? 🤔

Reply:
For the sliders in plot options, we can definitely control the throttling/debouncing, but that seems out of scope for what you're trying to do here.  In the viewer drag tool, we're attaching a callback to the viewer from the python side (rather than firing an event from Vue), and `viewer.add_event_callback` doesn't seem to accept any throttling options.

Reply:
So, are you saying this patch is okay as-is?

Reply:
I do think debouncing (in python still) will probably perform better than this discarding based on time, but that can be improved later if you'd rather.  It does look like JS-side handling isn't really feasible in this case (at least not without upstream changes to support it in `viewer.add_event_callback`).

Reply:
See #1403 for the debouncing implementation.